Keypoint3d Nose
Keypoint3d Neck
Keypoint3d RShoulder
Keypoint3d RElbow
Keypoint3d RWrist
Keypoint3d LShoulder
Keypoint3d LElbow
Keypoint3d LWrist
Keypoint3d RHip
Keypoint3d RKnee
Keypoint3d RAnkle
Keypoint3d LHip
Keypoint3d LKnee
Keypoint3d LAnkle
Keypoint3d REye
Keypoint3d LEye
Keypoint3d REar
Keypoint3d LEar
Keypoint3d Waist
